### DISCUSSION TITLE
Problems with page switching

### DISCUSSION THREAD

**Discussion by janneserichsen** [neuefische/questions#462] [Web]
I want to add a navbar what is allready done and linked but when i like to a other page than my starter page it switchs to the other page but shows the beginning pages content

**Comment by doemser** [comment]
To be able to help you here, I need several things from you:

- The affected code or a link to the affected, currently pushed Github Branch
- Screenshots showing the current state
- Error messages?

Also: What have you already tried? Have you already taken a look at the next/router Docs?

https://nextjs.org/docs/api-reference/next/router

**Reply by janneserichsen** [reply]
<img width="630" alt="Bildschirmfoto 2023-01-24 um 14 30 55" src="https://user-images.githubusercontent.com/118190990/214307785-1bb4ebb3-4847-4fa1-a956-2e41a13ecf54.png">

**Reply by janneserichsen** [reply]
no error messages

**Reply by janneserichsen** [reply]
but this comes up when i link to the current page

Error: Invariant: attempted to hard navigate to the same URL /info/index http://localhost:3000/info/index

**Reply by doemser** [reply]
Another nice to have would have been your pages folders structure, or you push your current state to branch and send this.

Right now it seems that you need to change the hrefs from "/info/index" to "/info" and "/profile/index" to "/profile"
